TrafficCapture/build.gradle

```diff
@@ -31,29 +33,38 @@ allprojects {
         systemProperty 'disableMemoryLeakTests', 'true'
         useJUnitPlatform {
             excludeTags 'longTest'
+            systemProperty 'junit.jupiter.execution.parallel.enabled', 'true'
+            systemProperty 'junit.jupiter.execution.parallel.mode.default', "concurrent"
+            systemProperty 'junit.jupiter.execution.parallel.mode.classes.default', 'concurrent'
+            systemProperty 'junit.jupiter.execution.parallel.config.strategy', 'dynamic'
         }
         jvmArgs '-ea'
 
         jacoco {
             enabled = false
         }
+        maxParallelForks = Runtime.runtime.availableProcessors().intdiv(2) ?: 1
```

Comment:
That was the example here: https://docs.gradle.org/current/userguide/performance.html#execute_tests_in_parallel

I can do some quick tests to see if the halving was beneficial

Reply:
Moved this over to the TrafficReplayer test, we don't want to fork anyways and instead rely on threads for junit parallelization except in that test suite since it isn't optimized for parallel execution